packages/fether-react/src/App/App.js

```diff
@@ -40,6 +45,55 @@ class App extends Component {
     electron.ipcRenderer.send('asynchronous-message', 'app-resize', height);
   };
 
+  state = {
+    newRelease: false // false | {name, url, ignore}
+  };
+
+  componentDidMount () {
+    window
+      .fetch('https://api.github.com/repos/paritytech/fether/releases/latest')
+      .then(j => j.json())
+      .then(({ name, html_url: url, tag_name: tag }) => {
+        const latestVersion = tag.match(/v(\d+\.\d+(\.\d+)?)/)[1];
+        if (semver.gt(latestVersion, currentVersion)) {
+          this.setState({
+            newRelease: {
+              name,
+              url,
+              ignore: false
+            }
+          });
+        }
+      })
+      .catch(e => {
+        console.error('Error while checking for a new version of Fether:', e);
+      });
+  }
+
+  renderModalLinks = () => {
+    return (
+      <nav className='form-nav -binary'>
+        <button className='button -back' onClick={this.hideNewReleaseModal}>
+          Remind me later
+        </button>
+
+        <button className='button' onClick={this.openNewReleaseUrl}>
+          Download
+        </button>
+      </nav>
+    );
+  };
+
+  hideNewReleaseModal = () => {
+    this.setState({
+      newRelease: { ...this.state.newRelease, ignore: true }
+    });
+  };
+
+  openNewReleaseUrl = () => {
+    window.open(this.state.newRelease.url);
```

Comment:
change external link to `window.open(this.state.newRelease.url, '_blank', 'noopener noreferrer');` for security (prevent new page from being able to access the window.opener property and ensure it runs in a separate process) and privacy protection (additionally prevents the Referer header from being sent to the new page)